tf/environments/production/main.tf

```diff
@@ -241,6 +243,68 @@ resource "aws_security_group" "clickhouse_sg" {
   tags = local.tags
 }
 
+### AWS RDS for PostgreSQL
+resource "aws_security_group" "pg_sg" {
+  description = "controls access to postgresql database"
+
+  vpc_id = aws_vpc.main.id
+  name   = "ooni-tier0-prod-postgres-sg"
+
+  ingress {
+    protocol    = "tcp"
+    from_port   = 5432
+    to_port     = 5432
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port = 0
+    to_port   = 0
+    protocol  = "-1"
+
+    cidr_blocks = [
+      "0.0.0.0/0",
+    ]
+  }
+
+  tags = local.tags
+}
+
+resource "aws_db_subnet_group" "main" {
+  name       = "ooni-main"
+  subnet_ids = [aws_subnet.main[0].id, aws_subnet.main[1].id]
+
+  tags = {
+    Name = "Main"
+  }
+}
+
+resource "aws_db_instance" "ooni_pg" {
```

Comment:
Do you mind prefixing a one-line comment to each stanza explaining what is going on?